src/issuance.rs

```diff
@@ -616,9 +607,10 @@ mod tests {
     use pasta_curves::pallas::{Point, Scalar};
     use rand::rngs::OsRng;
     use rand::RngCore;
-    use reddsa::Error::InvalidSignature;
     use std::collections::HashSet;
 
+    impl std::error::Error for Error {}
```

Comment:
This seems redundant together with `crate::issuance::{Error}

Reply:
Removed it, thanks.